app/app.go

```diff
@@ -390,7 +386,7 @@ func New(
 		govModuleAddr,
 		app.BankKeeper,
 		&app.HyperlaneKeeper,
-		[]int32{int32(warptypes.HYP_TOKEN_TYPE_COLLATERAL)},
+		[]int32{int32(warptypes.HYP_TOKEN_TYPE_COLLATERAL), int32(warptypes.HYP_TOKEN_TYPE_SYNTHETIC)},
```

Comment:
[question] should we add a unit test that confirms we can bridge synthetic tokens?

Reply:
It looks like how the synthetic tokens were tested earlier was via tests in the `test/interop` directory.

How about we modify this method in `test/interop`: https://github.com/celestiaorg/celestia-app/blob/6928c85dc740a858bf072e31cbb2048e7bbae88b/test/interop/hyperlane_test.go#L134

to check that synthetic tokens are enabled, would that be sufficient?

Reply:
Yup that would be good